storage/test/storage/ArrayListStorageTest.java

```diff
@@ -0,0 +1,15 @@
+package storage;
+
+import storage.list.ArrayListStorage;
+
+public class ArrayListStorageTest extends ArrayStorageTest {
+
+    public ArrayListStorageTest() {
+        super(new ArrayListStorage());
+    }
+
+    @Override
+    void save_throwsAddToFullStorageException() {
```

Comment:
LSP violation here. List is not an array with limits.

Reply:
Done